src/components/ContentSidebar/SidebarNotices.js

```diff
@@ -4,49 +4,73 @@
  * @author Box
  */
 
-import React from 'react';
+import React, { Fragment } from 'react';
```

Comment:
Just do `import * as React from 'react';` and then use <React.Fragment>
New-er adopted style.